microservices/dlstreamer-pipeline-server/utils/influx_client.py

```diff
@@ -10,6 +10,7 @@
 from influxdb_client.client.write_api import SYNCHRONOUS
 from src.publisher.influx.influx_schema import DataSchema
 from src.common.log import get_logger
+import urllib3
```

Comment:
hope this would not affect BoM, does it come natively with python

Reply:
urllib3 package is already present as part of the current image. It is not native but may be installed as part of the other package dependency.